src/_data/codebase/v2_3/mrg/page-builder/CatalogPageBuilderAnalytics.yml

```diff
@@ -0,0 +1,9 @@
+---
+title: Magento_CatalogPageBuilderAnalytics
+source_repo: magento2-page-builder
+release: v1.3.3
```

Comment:
Where is the "v" in the release property coming from (`release: v1.3.3`)? I don't see any other release properties for any other modules prepended with a "v". Does it matter?

Reply:
This property corresponds to a release tag as-is. That's how PageBuilder releases are tagged, differently from other repos (ce/ee/b2b/inventory).

Reply:
Since there is no single convention for tagging across different repositories. I used to add `v` during generation, but in case with PageBuilder it turned to `vv`. I decided to use it as-is to avoid any such issues in the future.

Reply:
Good to know, thanks.